fictadvisor-web/src/app/(main)/schedule/schedule-page/calendar-section/components/groups-dropdown/GroupsDropDown.tsx

```diff
@@ -15,23 +16,30 @@ export const GroupsDropDown: FC<DropDownSectionProps> = ({ groups }) => {
   }));
   const setGroupId = useSetGroupId();
 
+  const memoGroups = useMemo(() => {
+    return groups
+      ? groups?.map(group => ({
+          id: group.id,
+          label: group.code,
+        }))
+      : [];
+  }, [groups.length]);
+
+  const onChangeMemo = useCallback(
+    (id: string) => {
+      useSchedule.setState(state => ({ isNewEventAdded: false }));
```

Comment:
shouldn`t you call it like  (state => ({ ...state, isNewEventAdded: false }));?

Reply:
This is not my changes, someone did this in this way